pkg/apis/build/v1alpha2/build_pod.go

```diff
@@ -213,7 +212,7 @@ func (b *Build) BuildPod(images BuildPodImages, secrets []corev1.Secret, taints
 				step(corev1.Container{
 					Name:    "completion",
 					Image:   images.completion(config.OS),
-					Command: []string{"/cnb/process/web"},
+					Command: []string{fmt.Sprintf("/cnb/process/%s", b.DefaultProcess())},
```

Comment:
This is just the completion's image default process type. It should not be affected by the build's process.